src/sync-n-parent.js

```diff
@@ -0,0 +1,27 @@
+AFRAME.registerComponent('sync-n-parent',
```

Comment:
Good point, I'll see if I can separate the responsibilities here.

Since this component sets the userId property to the local user's id before it instantiates the component remotely, it only gives the item to the remote avatars of this user.